Zend/zend_alloc.c

```diff
@@ -2945,6 +2958,16 @@ ZEND_API zend_mm_heap *zend_mm_get_heap(void)
 	return AG(mm_heap);
 }
 
+ZEND_API zend_mm_heap *zend_mm_heap_create(void)
+{
+	return zend_mm_init();
+}
+
+ZEND_API void zend_mm_heap_free(zend_mm_heap *heap)
+{
+	zend_mm_chunk_free(heap, heap->main_chunk, ZEND_MM_CHUNK_SIZE);
+}
+
```

Comment:
I didn't understand the meaning of this API call.
``heap->main_chunk`` should be freed by ``zend_mm_shutdown`` when ``full`` argument is ``true``,

Reply:
You are right, I can just use the `zend_mm_shutdown` to achieve the same effect.